examples/single-module-kotlin-dsl-gradle/src/main/java/com/example/HelloWorld.java

```diff
@@ -0,0 +1,9 @@
+package com.example;
+
+public class HelloWorld {
+
+    public static void main(String[] args) {
```

Comment:
Hi @devtribe,
thanks for your PR. I think your example is `java`-code. Can you rewrite this to `kotlin`?
Best Regards
Reamer

Reply:
Hi @Reamer,

sure i can do this, but the code of the application is not really important, the important thing is the Gradle Kotlin DSL in die `build.gradle.kts`.
Just give me a ping if i should rewrite it.

Greetings

Reply:
Yes please rewrite. Our gradle kotlin example application should be written in kotlin.